cypress/integration/plugins/security/permissions_spec.js

```diff
@@ -104,6 +104,7 @@ if(Cypress.env("SECURITY_ENABLED")) {
       cy.contains('button', 'Create action group').first().click({ force: true });
       cy.get('button[id="create-from-selection"]').should('have.attr', 'disabled');
   
+      cy.wait(500);
```

Comment:
Can we create a follow up issue to deal with the arbitrary wait time [[doc](https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting)].